yarn.lock

```diff
@@ -7769,7 +7769,7 @@ file-type@^6.1.0:
   resolved "https://registry.yarnpkg.com/file-type/-/file-type-6.2.0.tgz#e50cd75d356ffed4e306dc4f5bcf52a79903a919"
   integrity sha512-YPcTBDV+2Tm0VqjybVd32MHdlEGAtuxS3VAYsumFokDSMG+ROT5wawGlnHDoz7bfMcMDt9hxuXvXwoKUx2fkOg==
 
-filelist@^1.0.1:
+filelist@^1.0.4:
```

Comment:
This happened just on a `yarn install` before I made any changes to resolutions.